addons/binding/org.openhab.binding.tplinksmarthome/README.md

```diff
@@ -181,7 +181,7 @@ tplinksmarthome:lb130:home "Living Room Bulb 2" [ ipAddress="192.168.0.15", refr
 Switch   TP_L_Switch  "Switch"                            { channel="tplinksmarthome:hs100:home:switch" }
 Number   TP_L_RSSI    "Signal [%d] dB"           <signal> { channel="tplinksmarthome:hs100:home:rssi" }
 Dimmer   TP_LB_Bulb   "Dimmer [%d %%]"           <slider> { channel="tplinksmarthome:lb110:home:brightness" }
-Dimmer   TP_LB_ColorT "Color Temperature [%d] K" <slider> { channel="tplinksmarthome:lb130:home:color" }
+Dimmer   TP_LB_ColorT "Color Temperature [%d] %%" <slider> { channel="tplinksmarthome:lb130:home:colorTemperature" }
```

Comment:
Was it already mentioned in the XML?

Reply:
During initial commit at some point the binding had a channel `colorTemperatureKelvin`. On this channel you could set color Temperature in Kelvin. There is also a color Temperature channel that works with percentages. Upon code review by @kaikreuzer this Kelvin channel was removed. I just noticed the readme contained a typo that can be traced back to this earlier implementation, and was incorrectly changed when removing that kelvin channel.